### PR TITLE
WD-5128 - Build the ai/roadshow page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done

- Build the ai/roadshow page

## QA

- Open the demo
- Go to /ai/roadshow
- Check the copy matches the [copy doc](https://docs.google.com/document/d/1CA0GUU6yRB65yfDaKi1dgN8DED2bDAFI2h-swSVtMtg/edit)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-5128

### CODE REVIEW

File: templates/ai/roadshow.html [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="p-block">
+          <hr class="p-rule" />
+          <div class="row">
+            <div class="col-3 col-medium-2 col-small-2">
+              <h3 class="p-heading-5">Meet AI/ML experts and ask your questions</h3>
```
Comment (lyubomir-popov) [inline templates/ai/roadshow.html:61]:
```suggestion
              <h3 class="p-heading--5">Meet AI/ML experts and ask your questions</h3>
```

File: templates/ai/roadshow.html [hunk 2]
```diff
@@ -0,0 +1,390 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="p-block">
+          <hr class="p-rule" />
+          <div class="row">
+            <div class="col-3 col-medium-2 col-small-2">
+              <h3 class="p-heading-5">See our demos and try building your own LLM</h3>
```
Comment (lyubomir-popov) [inline templates/ai/roadshow.html:76]:
```suggestion
              <h3 class="p-heading--5">See our demos and try building your own LLM</h3>
```

File: templates/ai/roadshow.html [hunk 3]
```diff
@@ -0,0 +1,390 @@
...
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-start-large-4">
+        <div class="p-block">
+          <hr class="p-rule" />
+          <div class="row">
+            <div class="col-3 col-medium-2 col-small-2">
+              <h3 class="p-heading-5">Run AI projects in production securely</h3>
```
Comment (lyubomir-popov) [inline templates/ai/roadshow.html:92]:
```suggestion
              <h3 class="p-heading--5">Run AI projects in production securely</h3>
```

**Review by lyubomir-popov — APPROVED** [review]

**Comment by lyubomir-popov** [comment]
Looks good, I'd run these 2 sections past @Sophie-32, and maybe wrap the bottom in a deep strip - it is next to the footer so the more separation the better:

image

image

**Comment by lyubomir-popov** [comment]
Sorry last thing, can we match the original aspect ratio to prevent clipping on the images:
image

**Comment by juanruitina** [comment]
Copy wasn't edited, I made a few adjustments in the copy doc. Sophie might have more feedback when she comes back on Monday.

For now:

- [ ] Make "Would you like to have us at your event?..." and "You can also follow us on [Medium](http://medium.com/ubuntu-ai)" regular paragraphs (keep the styling).
- [ ] Add "Webinar" and "Case study" with `p-text--small-caps` styling above each resource's title (added to the copy doc), similar to "Joint webinar" [here](https://ubuntu.com/hpe).
- [ ] Link the resource's titles and thumbnails to the resource (added to the copy doc)
- [ ] Please make sure to separate the talks under "World Summit AI" and "MLOps World" as per the copy doc, maybe a separate paragraph will suffice.

**Comment by anthonydillon** [comment]
> Make "Would you like to have us at your event?..." and "You can also follow us on [Medium](http://medium.com/ubuntu-ai)" regular paragraphs (keep the styling).

What do you mean? Make them paragraphs as in 16px?

**Comment by juanruitina** [comment]
@anthonydillon I mean we should use for them a `p` element with the `p-heading--2` class, to keep a clean heading/accessibility tree while keeping the styling, which is correct.

**Comment by anthonydillon** [comment]
@juanruitina addressed your comments

**Comment by Sophie-32** [comment]
I added some copy changes to the copy doc, Andreea will review shortly

**Comment by juanruitina** [comment]
We've reviewed the [copy doc](https://docs.google.com/document/d/1CA0GUU6yRB65yfDaKi1dgN8DED2bDAFI2h-swSVtMtg/edit), can you please make sure the copy matches it?

We have also changed how we present the two resources at the end of the page to use cards, [see here in the wireframe](https://www.figma.com/file/Luq6Dr26oX80421NDPljPU/23.10-U.com---AI-Roadshow-(WD-949)?type=design&node-id=1%3A3&mode=design&t=z2d2ADTgUisAVgsO-1). This is similar to what we did for joint webinars on the [Dell page](https://ubuntu.com/dell), in case you want to reuse that code.

Also, we are fine with including "Roadshow" in the bubble navigation, as Andreea requested.

**Review by carkod — APPROVED** [review]
LGTM

**Comment by anthonydillon** [comment]
@juanruitina @lyubomir-popov could you review this again please?

**Comment by lyubomir-popov** [comment]
Looks good. Can you please update to the latest vanilla? We now have ampersands (and other numbers/symbols) matching the small caps, this is what it would look like (notice the ampersand no longer sticks out):
image

**Comment by anthonydillon** [comment]
That would be far outside the scope of this branch.

**Comment by juanruitina** [comment]
Three minor comments:

- Change the styling of the subtitle ("Hear about production-grade LLM...") to `p-heading--5` styling.
- To keep the heading tree clean, please make the event titles ("World Summit AI", "Ubuntu Summit"...) `<h4>` elements while keeping the current styling; and titles under "AI/ML resources" ("A guide to MLOps", "University of Tasmania unlocks real-time space tracking with AI/ML supercomputing") `<h3>`, again keeping the styling.
- The two big CTA sections ("To join us at an event, book a meeting", "Follow us on Medium") should have a white background instead of the current light gray one.

But it generally LGTM. Feel free to change the label to UX+1 when you address this.

**Comment by anthonydillon** [comment]
Thanks, @juanruitina addressed and ready for review.

**Comment by juanruitina** [comment]
Looks great!

**Pull request merged**